docker-compose.yml

```
@@ -0,0 +1,38 @@
+version: '3.6'
+services:
+
+  app:
+    build: .
+    container_name: easy-lives-app
+    command: rails server --port 3000 --binding 0.0.0.0
```

```suggestion
    command: bin/rails server --port 3000 --binding 0.0.0.0
```
just to follow Rails docs recommendation :)